pkgs/by-name/ch/check-sieve/package.nix

```
@@ -0,0 +1,60 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  bison,
+  diffutils,
+  flex,
+  python3,
+  nix-update-script,
+}:
+stdenv.mkDerivation rec {
+  pname = "check-sieve";
+  version = "0.10";
+
+  src = fetchFromGitHub {
+    owner = "dburkart";
+    repo = "check-sieve";
+    rev = "check-sieve-${version}";
```

```suggestion
    tag = "check-sieve-${version}";
```

(Make sure you are on latest checkout, rebase if not)

---

I replaced it with `tag`, although it looks like nix-update currently doesn't work with `tag` instead of `rev`